src/howl/core.cljc

```diff
@@ -211,8 +211,8 @@
     PREDICATE  = PREFIXED_NAME / WRAPPED_IRI / ABSOLUTE_IRI / LABEL
     OBJECT     = BLANK_NODE / PREFIXED_NAME / WRAPPED_IRI / ABSOLUTE_IRI / LABEL
     DATATYPE   = PREFIXED_NAME / WRAPPED_IRI / ABSOLUTE_IRI / LABEL
-    LITERAL    = #'.+(?=@(\\w|-)+)' LANG /
-                 #'.+(?=\\^\\^\\S+)' '^^' DATATYPE /
+    LITERAL    = CHAR+ LANG /
+                 CHAR+ <'^^'> DATATYPE /
```

Comment:
Lets keep it in for the moment; we're going to be restructuring this part of the parser shortly anyhow.